net/knot/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=knot
-PKG_VERSION:=2.7.6
+PKG_VERSION:=2.8.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://secure.nic.cz/files/knot-dns/
-PKG_HASH:=a1cb1877f04f7c2549c977c2658cfafd07c7e0e924f8e8aa8d4ae4b707f697a2
+PKG_MD5SUM:=494ad926705018bd754d96711dc2129f3173f326a0b57d33978090ba4eef87ef
```

Comment:
Wrong.
MD5SUM is the deprecated variable. And looks like you are placing SHA256 hash there in any case (just like you should...)

PKG_HASH is the current variable.

Reply:
I'm sorry. It was a typo.